src/Platform.Handlers/src/Xamarin.Platform.Handlers/Platform/iOS/ViewExtensions.cs

```diff
@@ -22,7 +22,7 @@ public static void UpdateBackgroundColor(this UIView nativeView, IView view)
 
 			var color = view.BackgroundColor;
 
-			if (color != null && !color.IsDefault)
+			if (!color.IsDefault)
```

Comment:
Nullability checks inside NET6 throw a warning here because color will always be not null